src/styles/page-template/custom/code.njk

```diff
@@ -16,9 +16,11 @@ ignore_in_sitemap: true
 
 {% set htmlClasses = "app-html-class" %}
 {% set htmlLang = "en" %}
+{% set pageTitleLang = "cy" %}
```

Comment:
Sorry to be annoying since I suggested this but I think by changing these and not changing the content this is creating an example that is incorrect.

So maybe:

- I was wrong to suggest this
- or we should include them but set them to 'en'
- or we should comment them out

Reply:
Yeah it's tricky, I was thinking that setting them to `cy` would demonstrate that they can (and should be) different from the main page language. But that's explained above so maybe that covers it. I think there isn't an expectation that all the values set in the Customised Page template section would necessarily make sense when put together. 

Have changed them to `en` - will get some content feedback on this too.

Reply:
I'm a little unsure about this – mainly because I don't think we would want people to put lang="en" on the page title or main block if the html lang is "en" anyway.

How important is it that we showcase every single thing you can override in this example? If we think this is important enough, should we have a separate example that just shows how to handle different languages on the same page in a realistic context?

Reply:
I think it's probably important to show case the things that people are likely to want to change, this seems like it would not be something most people are likely to change soon so I'd be OK leaving these out.

Reply:
Okay have taken them out 👍